ml-agents-envs/mlagents_envs/communicator_objects/unity_to_external_pb2_grpc.py

```diff
@@ -1,46 +1,67 @@
 
-    Args:
-      channel: A grpc.Channel.
-    """
-    self.Exchange = channel.unary_unary(
-        '/communicator_objects.UnityToExternalProto/Exchange',
-        request_serializer=mlagents__envs_dot_communicator__objects_dot_unity__message__pb2.UnityMessageProto.SerializeToString,
-        response_deserializer=mlagents__envs_dot_communicator__objects_dot_unity__message__pb2.UnityMessageProto.FromString,
-        )
+        Args:
+            channel: A grpc.Channel.
+        """
+        self.Exchange = channel.unary_unary(
+                '/communicator_objects.UnityToExternalProto/Exchange',
+                request_serializer=mlagents__envs_dot_communicator__objects_dot_unity__message__pb2.UnityMessageProto.SerializeToString,
+                response_deserializer=mlagents__envs_dot_communicator__objects_dot_unity__message__pb2.UnityMessageProto.FromString,
+                )
 
 
 class UnityToExternalProtoServicer(object):
-  # missing associated documentation comment in .proto file
-  pass
+    """Missing associated documentation comment in .proto file."""
 
-  def Exchange(self, request, context):
-    """Sends the academy parameters
-    """
-    context.set_code(grpc.StatusCode.UNIMPLEMENTED)
-    context.set_details('Method not implemented!')
-    raise NotImplementedError('Method not implemented!')
+    def Exchange(self, request, context):
+        """Sends the academy parameters
+        """
+        context.set_code(grpc.StatusCode.UNIMPLEMENTED)
+        context.set_details('Method not implemented!')
+        raise NotImplementedError('Method not implemented!')
 
 
 def add_UnityToExternalProtoServicer_to_server(servicer, server):
-  rpc_method_handlers = {
-      'Exchange': grpc.unary_unary_rpc_method_handler(
-          servicer.Exchange,
-          request_deserializer=mlagents__envs_dot_communicator__objects_dot_unity__message__pb2.UnityMessageProto.FromString,
-          response_serializer=mlagents__envs_dot_communicator__objects_dot_unity__message__pb2.UnityMessageProto.SerializeToString,
-      ),
-  }
-  generic_handler = grpc.method_handlers_generic_handler(
-      'communicator_objects.UnityToExternalProto', rpc_method_handlers)
-  server.add_generic_rpc_handlers((generic_handler,))
+    rpc_method_handlers = {
+            'Exchange': grpc.unary_unary_rpc_method_handler(
+                    servicer.Exchange,
+                    request_deserializer=mlagents__envs_dot_communicator__objects_dot_unity__message__pb2.UnityMessageProto.FromString,
+                    response_serializer=mlagents__envs_dot_communicator__objects_dot_unity__message__pb2.UnityMessageProto.SerializeToString,
+            ),
+    }
+    generic_handler = grpc.method_handlers_generic_handler(
+            'communicator_objects.UnityToExternalProto', rpc_method_handlers)
+    server.add_generic_rpc_handlers((generic_handler,))
+
+
+ # This class is part of an EXPERIMENTAL API.
+class UnityToExternalProto(object):
```

Comment:
Did you expect this file to change (especially with this new class)?

Reply:
This is an artifact of my building the proto files using a newer version of `grpc-tools`, as I am still unable to install older versions. I'll try again to figure out the problem and build these with the version we are using internally though, since there are obviously discrepancies in what is being produced.